dockers/skylab/snss2-build-indices/add-introns-to-gtf.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python3
```

Comment:
Is a .py a docker? should this be in the scripts folder?

Reply:
All dockers in skylab have their scripts next to then if they have scripts in the docker. It makes it easier to change the docker in case we want to modify those scripts.

Reply:
Ok, thanks